src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/import_features_random_conditions.json

```diff
@@ -21,8 +21,8 @@
           }
         }
       ],
-      "requirement_type": "All",
-      "random_condition": "random_value"
+      "RequirementType": "All",
```

Comment:
Could you add a new test for validating RequirementType?

Reply:
New tests have been added to validate the "Any/All" values when importing feature flags